src/components/calcite-date-picker-day/calcite-date-picker-day.scss

```diff
@@ -38,6 +39,7 @@
     @apply p-0;
   }
   .day {
+    //FIXME: This needs to be refactored by design first
```

Comment:
these are odd values so that text can be perfectly centered inside, I believe

Reply:
okay, I'll remove comment and leave as is for now.